### PR TITLE
Setting the default PR merge policy to 'squash'.

### PR DESCRIPTION
**Pull request by ybettan**
It means that every PR will be squashed to a single commit before
merging regardless of the number of commits in the PR pre-merging.

Signed-off-by: Yoni Bettan <yonibettan@gmail.com>

### CODE REVIEW

**Comment by ybettan** [comment]
/cc @qbarrand 
/hold
Do not merge before https://github.com/kubernetes-sigs/kernel-module-management/pull/9

**Comment by qbarrand** [comment]
/ok-to-test
/hold
Hold until https://github.com/kubernetes-sigs/kernel-module-management/pull/9 is merged, for which we need individual commits.

**Comment by ybettan** [comment]
Hello @cblecker and @wojtek-t.
Can one of you please `lgtm`?

File: config/prow/config.yaml [hunk 1]
```diff
@@ -777,6 +777,7 @@ tide:
     kubernetes-sigs/go-open-service-broker-client: squash
     kubernetes-sigs/minibroker: squash
     kubernetes-sigs/vsphere-csi-driver: squash
+    kubernetes-sigs/kernel-module-management: squash
```
Comment (qbarrand) [inline config/prow/config.yaml:780]: Can we keep an alphabetical order? I would put this line just above `kubernetes-sigs/krew-index`.

Comment (ybettan) [reply config/prow/config.yaml:780]: Sure. Fixing

Comment (ybettan) [reply config/prow/config.yaml]: Fixed.

**Pull request merged**